experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -243,16 +246,17 @@ export class NodeSDK {
    * Create a new NodeJS SDK instance
    */
   public constructor(configuration: Partial<NodeSDKConfiguration> = {}) {
-    if (getBooleanFromEnv('OTEL_SDK_DISABLED')) {
+    const configFactory: ConfigFactory = createConfigFactory();
```

Comment:
Re-reviewing the `createConfigFactory`, I think we need to revisit `hasValidConfigFile()`. If I am reading correctly, it will throw an error if a user provides an invalid error to the value for `OTEL_EXPERIMENTAL_CONFIG_FILE`. So if someone happens to do this incorrectly it will crash their app right? We should probably consider logging an error instead of throwing an error so the overall impact is minimal if the config value is not valid.

Reply:
I went back and forward on this, My decision was to make it an error  since this file will define pretty much everything for the instrumentation, if it's wrong I wanted to make it very visible, otherwise just a warning could be missed and all the instrumentation being setup could be incorrect.

I don't have a strong opinion and I can change the behaviour, just wondering how we can make this very visible. If we think that a warning is enough, that's okay :)

Reply:
PR with the changes: https://github.com/open-telemetry/opentelemetry-js/pull/6124